### PR TITLE
LPD-31633 Check if Item Selectors Browser Breadcrumb is updated after…

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi,
I would like to kindly ask to review my PR
This is opening again from:
https://github.com/liferay-content-management/liferay-portal/pull/5743
Thank you in advance!
BR

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:sf

**Review by ambrinchaudhary — COMMENTED** [review]
Missing `test.properties` in `modules/apps/item-selector/item-selector-taglib`. 
Also, it is really necessary the change in `modules/apps/item-selector/test.properties`? Other modules don't have it...

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
Thanks @ambrinchaudhary !

**Comment by gergelyszalay** [comment]
ci:test:playwright

File: test.properties [hunk 1]
```diff
@@ -4323,6 +4323,7 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         blogs-web,\
         content-dashboard-web,\
         document-library-web,\
+        item-selector,\
```
Comment (ambrinchaudhary) [inline test.properties:4326]: I think this should be `item-selector-taglib`, isn't it?

**Comment by gergelyszalay** [comment]
ci:test:stop

**Comment by gergelyszalay** [comment]
ci:test:sf

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**